fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -18,6 +18,8 @@
  * limitations under the License.
  */
 
+#include "fdbclient/FDBTypes.h"
+#include <variant>
```

Comment:
Nit: put these below the `#define`. What types in `FDBTypes.h` are being used here? I only ask because this header is included everywhere and changing it already makes the recompile take a long time.

Reply:
Removed `FDBTypes.h` import (was added automatically by clang format)